cluster/interface.go

```diff
@@ -7,15 +7,15 @@ import (
 const (
 	// EtcdLeaderElector is the id used when constructing an
 	// etcdLeaderElector instance through the factory.
-	EtcdLeaderElector = "etcdLeaderElector"
+	EtcdLeaderElector = "etcd"
```

Comment:
nit: move these two changes in this file to the previous commit?

Reply:
done